lessons/Early-Math/distributive-property/langs/en-us/index.adoc

```diff
@@ -119,46 +153,135 @@ Some students may suggest that distributing is a way of addressing the grouping
+
+@slidebreak
 
 @lesson-instruction{
-- Turn to @printable-exercise{pages/mental-math.adoc} and look at the first problem.
-- Our goal here is to make the math easier by creating an equivalent expression that we can solve in our heads! So, instead of finding the product of 70 and 39, we are going to multiply 70 by the difference of 40 and 1.
 - Complete the next Circle of Evaluation, which shows that we are going to _distribute_ 70.
-- We can now compute our solution by finding difference between two products.
-- Do you find this process more efficient than the standard algorithm for multiplication? Explain.
-  * Note: Without ample practice, there is a good chance that students will not find this process efficient. Like any solving strategy, it takes practice!_
+- Observe how we can now compute our solution by finding the difference between two products.
+}
+
+@QandA{
+@Q{Do you find this process more efficient than the standard algorithm for multiplication? Explain.}
+}
+
+@lesson-instruction{
 - Complete the rest of the page by creating equivalent expressions that we can solve in our heads.
 }
 
+
+@teacher{Without ample practice, there is a good chance that students will not find this process efficient. Like any solving strategy, it takes practice!}
+
 === Synthesize
 
 - How can you multiply two 2-digit numbers using mental math?
-- Can you think of a multiplication problem that would be _easier_ to solve using the Distributive Property?
 - What sorts of problems are simpler to compute using the Distributive Property?
+- Can you think of a multiplication problem that would be _easier_ to solve using the Distributive Property?
+
 
 @teacher{We recommend inviting many students to share the problems they come up with. Make a list on the board for students to review together.}
+
+== Programming Exploration: Distribution @duration{20 minutes}
+
+=== Overview
+
+Extending concepts explored earlier in the lesson, students consider whether various functions that we use when coding are distributive.
+
+=== Launch
+
+We've learned a lot about how the distributive property works. We know that multiplication can distribute over addition or subtraction, allowing us to complete otherwise complex computations with efficiency and ease.
+
+@slidebreak
+
+But what about functions that deal with images? For instance, does `scale` distribute over `beside`?!
+
+@ifnotslide{
+[.embedded, cols="^.^4,^.^1,^.^4", stripes="none", grid="none", frame="none"]
+|===
+
+| @show{(coe '(scale 0.5 (beside aqua-star orange-dot)))} | &rarr; | @show{(coe  '(beside (scale 0.5 orange-dot) (scale 0.5 aqua-star)))}
+
+|===
+}
+
+@lesson-instruction{
+- After studying the two Circles of Evaluation (above), make a prediction: What image will the second Circle of Evaluation produce? Draw your prediction on @printable-exercise{distribution-and-code.adoc}.
+- Test your prediction. Type into @starter-file{editor} the code for the second Circle of Evaluation.
```

Comment:
You love this grammar!

I'd really recommend changing it to "Type the code for second Circles of Evaluation into the @starter-file{editor}"

Reply:
omg! Keep pointing it out, I'll eventually stop doing it!